cfgov/paying_for_college/disclosures/scripts/notification_tester.py

```diff
@@ -18,53 +18,62 @@
+    Send fake notifications to school endpoints to help troubleshoot.
 
-    You can send a test notification to a school's endpoint to confirm
+    You can send test notifications to school endpoints to confirm
     reception, or send to httpbin to check the outbound IP (origin),
     headers, and the data payload.
     Examples:
-    - print(send_test_notification(EDMC_PROD, OID, ERRORS))
-    - print(send_test_notification(BPI_PROD, OID, ERRORS))
-    - print(send_test_notification(BINPOST, OID, ERRORS))
+    - print(send_test_notifications())
+    - print(send_test_notifications(url=BINPOST))
     """
     prod_endpoints = {
         'edmc': Contact.objects.get(name='EDMC').endpoint,
         'bpi': Contact.objects.get(name='Bridgepoint Education').endpoint
     }
-    if url in prod_endpoints:
-        _url = prod_endpoints[url]
+    if not url:
+        urls = list(prod_endpoints.values())
     else:
-        _url = url
+        urls = [url]
     payload = {
         'oid': oid,
-        'time': "{0}+00:00".format(datetime.datetime.now().isoformat()),
+        'time': f"{datetime.datetime.now().isoformat()}+00:00",  # noqa
         'errors': errors
     }
-    try:
-        resp = requests.post(_url, data=payload, timeout=10)
-    except requests.exceptions.ConnectTimeout:
-        return "post to {} timed out".format(url)
-    if not resp.content:
-        print("Got a blank response from {}".format(url))
-        return
-    json_content = resp.json()
-    report = REPORT.format(
-        url,
-        resp.ok,
-        resp.reason,
-        resp.status_code,
-        payload['time'],
-        json_content.get('headers'),
-        json_content.get('origin')
-    )
-    return report
+    msg = ""
+    for _url in urls:
+        try:
+            resp = requests.post(_url, data=payload, timeout=10)
+        except requests.exceptions.ConnectTimeout:
+            msg += f"Post to {_url} timed out.\n"
+        else:
+            report = REPORT.format(
+                _url,
+                resp.ok,
+                resp.reason,
+                resp.status_code,
+                payload['time'],
+            )
+            msg += f"{report}\n"
+            if resp.content:
+                msg += f"Response content: {resp.content}\n"
+    return msg
 
-# curl test example:
-# curl -v -X POST --data "oid=f38283b5b7c939a058889f997949efa566c616c5&errors=INVALID: test notification via curl&time=2016-01-21T18:36:09.922690+00:00" --url "https://dev.exml.edmc.edu/cfpb"  # noqa
+
+# curl_examples
+"""
+curl -v https://httpbin.org/get
+curl -v -X POST https://httpbin.org/post
+
+export XDATA="oid='f38283b5b7c939a058889f997949efa566c616c5'&errors='INVALID: test notification via curl'&time='2020-06-24T18:36:09.922690+00:00'". # noqa
+export BPI="https://sissecureservices.bpiedu.com/routingmanager/api/cfpb"
+export EDMC="https://exml.dcedh.org/cfpb"
+
+curl -v -X POST --data "$XDATA" --url "$BPI"
+curl -v -X POST --data "$XDATA" --url "$EDMC"
```

Comment:
```suggestion
curl -v --data "$XDATA" --url "$BPI"
curl -v --data "$XDATA" --url "$EDMC"
```

Minor: `-X POST` is not needed if you use `--data` as it POSTs by default